[FAB-17265] Move private data tutorial to use test network (#457)
hyperledger/fabric: docs/source/private_data_tutorial.rst

shown later in the tutorial.

This collection definition file is deployed when the chaincode definition is

> committed to the channel using the `peer lifecycle chaincode commit command <file:///Users/nikhilgupta/fabric/docs/build/html/commands/peerlifecycle.html#peer-lifecycle-chaincode-commit>`__.

Reviewer: Looks like you are referencing a file on your system rather than the web link?

> committed to the channel using the `peer lifecycle chaincode commit command <file:///Users/nikhilgupta/fabric/docs/build/html/commands/peerlifecycle.html#peer-lifecycle-chaincode-commit>`__.

Reviewer: ```suggestion
committed to the channel using the `peer lifecycle chaincode commit command <http://hyperledger-fabric.readthedocs.io/en/latest/commands/peerlifecycle.html#peer-lifecycle-chaincode-commit>`__.
```

Author: 😬 Looks like I got the link from the wrong browser window. Thanks for the catch.

:guilabel:`Try it yourself`

Before installing, defining, and using the marbles private data chaincode below,
we need to start the Fabric test network. For the sake of this tutorial, we want
to operate from a known initial state. The following command will kill any active

> or stale docker containers and remove previously generated artifacts.

Reviewer: ```suggestion
or stale Docker containers and remove previously generated artifacts.
```

Author: Done, found somewhere else it was lowercase...

   docker rm -f $(docker ps -a | awk '($2 ~ /dev-peer.*.marblesp.*/) {print $1}')
   docker rmi -f $(docker images | awk '($1 ~ /dev-peer.*.marblesp.*/) {print $3}')

From the ``test-network`` directory, you can use the following command to start
up the Fabric test network with CouchDB:

.. code:: bash

   ./network.sh up createChannel -s couchdb

This command will deploy a Fabric network consisting of a single channel named

> ``mychannel`` with two organizations (each maintaining two peer nodes) and an

Reviewer: ```suggestion
``mychannel`` with two organizations (each maintaining one peer node) and an
```
I think there's only one peer per org?

Author: yes. done

    2019-04-22 19:09:04.336 UTC [cli.lifecycle.chaincode] submitInstallProposal -> INFO 001 Installed remotely: response:<status:200 payload:"\nKmarblespv1:57f5353b2568b79cb5384b5a8458519a47186efc4fcadb98280f5eae6d59c1cd\022\nmarblespv1" >
    2019-04-22 19:09:04.336 UTC [cli.lifecycle.chaincode] submitInstallProposal -> INFO 002 Chaincode code package identifier: marblespv1:57f5353b2568b79cb5384b5a8458519a47186efc4fcadb98280f5eae6d59c1cd

> 3. Now use the CLI operate as the Org2 admin. Copy and paste the following block of

Reviewer: ```suggestion
3. Now use the CLI operating as the Org2 admin. Copy and paste the following block of
```

or

```suggestion
3. Now use the CLI as the Org2 admin. Copy and paste the following block of
```

Author: done.

commands as a group and run them all at once:

.. code:: bash

    export CORE_PEER_LOCALMSPID="Org2MSP"
    export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
    export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp
    export CORE_PEER_ADDRESS=localhost:9051

> 4. Run the following command install the chaincode on the Org2 peer:

Reviewer: ```suggestion
4. Run the following command to install the chaincode on the Org2 peer:
```

Author: done.

:guilabel:`Try it yourself`

> Switch back Org1 using the following commands. Copy and paste the following code

Reviewer 2: Switch back to Org1

Author: done.
